ocs_ci/ocs/ui/views.py

```diff
@@ -1824,7 +1824,8 @@
     ),
     "select_administrator": (
         "//a[@class='pf-c-dropdown__menu-item']//h2[@class='pf-c-title pf-m-md'][normalize-space()='Administrator'] | "
-        "//a[@class='pf-m-icon pf-v5-c-dropdown__menu-item']//h2[normalize-space()='Administrator']",
+        "//a[@class='pf-m-icon pf-v5-c-dropdown__menu-item']//h2[normalize-space()='Administrator'] | "
+        "//h2[.='Administrator']",
```

Comment:
hello @ebondare, just doublechecking: are you sure, that `//h2[.='Administrator'] ` is selecting a dropdown item but will not select a placeholder of the dropdown.

Reply:
Thank you, fixed